cmd/oy-periodic-queries/main.go

```diff
@@ -0,0 +1,82 @@
+// Copyright The o11y toolkit Authors
+// spdx-license-identifier: apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"os"
+
+	"github.com/go-kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/o11ydev/oy-toolkit/util/client"
+	"github.com/o11ydev/oy-toolkit/util/cmd"
+	"github.com/o11ydev/oy-toolkit/util/http"
+
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+)
+
+func main() {
+	ruleFiles := kingpin.Arg("rule-file", "File to read periodic recording rules from.").Required().Strings()
+	prefetch := kingpin.Flag("prefetch", "Cache metrics before exposing them. Avoid scrape timeout.").Default("true").Bool()
+	c := client.InitCliFlags()
+	logger := cmd.InitCmd("oy-periodic-files")
```

Comment:
This causes the output of the command to be, probably typoed `files` here instead of `queries`?

```
$ go run ./cmd/oy-periodic-queries/ --version                                                                                                                                                                                              ↵ 1
oy-periodic-files, version  (branch: , revision: unknown)
  build user:
  build date:
  go version:       go1.20
  platform:         linux/amd64
```